src/main/java/nl/hsac/fitnesse/fixture/util/JsonPathHelper.java

```diff
@@ -54,4 +51,12 @@ public Object getJsonPath(String json, String jsonPath) {
         }
         return result;
     }
+
+    public String updateJsonPathWithValue(String json, String jsonPath, String value) {
+        if(null != getJsonPath(json, jsonPath)) {
```

Comment:
This means you can't set a value for a field not previously present. Is that a requirement/good idea?
This may be an acceptable limitation for now, but just want to check.